Makefile

```diff
@@ -274,19 +259,32 @@ test-docker-push: test-docker
 	docker-build-debug docker-build-hermes docker-build-all
 
 
+
+
 ###############################################################################
 ###                                Protobuf                                 ###
 ###############################################################################
+
+protoImageName=ghcr.io/faddat/proto-builder:latest
```

Comment:
What's this?

Reply:
it is the custom proto builder that I made for this.  Want it in the repo instead?

We could do like:

* merge this with the ci system from github.com/faddat/proto-builder
  * gaia will then have a proto builder in her repo in ghcr (github packages / docker registry)
* make a second pr to use that instead of the one in faddat/proto-builder

Reply:
this is resolved here:

* #2506